### PR TITLE
add Svelte workshop

### PR DESCRIPTION
**Pull request by marcoow**
## TODO

- [x] add `og:image`
- [x] change header image
- [x] add Gabor's and Dan's bios

### CODE REVIEW

**Comment by marcoow** [comment]
@davidhajba can you create an `og:image` as well as a header image? @beerinho @zeppelin @pichfl @paoloricciuti do you have ideas for a good header image?

File: src/workshops/svelte-and-sveltekit.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    text: >
+      SvelteKit comes with two testing strategies: Vitest for testing components
+      in isolation and Playwright for end-to-end testing. We cover both in depth
+      and present approaches for testing real applications.
+  - heading: Stores
+    text: >
+      This stage kicks off with an introduction of what stores are and how they
+      work. We continue with looking into implementing the three main stores:
+      writable, readable, and derived.
+  - heading: Form Actions
```
Comment (beerinho) [inline src/workshops/svelte-and-sveltekit.md]:
```suggestion
  - heading: SvelteKit as a server
```

We updated this heading as this section wasn't just about form actions

File: src/workshops/svelte-and-sveltekit.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    bio: >
+      …
+  - name: Florian Pichler
+    title: Senior Frontend Engineer
+    handle: pichfl
+    authorHandle: pichfl
+    image: /assets/images/authors/pichfl.jpg
+    bio: >
+      Florian has been solving frontend challenges for over 15 years, with all
+      tools that became available. He built and coached teams of all sizes.
```
Comment (beerinho) [inline src/workshops/svelte-and-sveltekit.md]:
```suggestion
      tools that became available. He has built and coached teams of all sizes.
```

**Review by beerinho — COMMENTED** [review]
Looks great in general, just a couple of minor tweaks to the wording

**Comment by beerinho** [comment]
I think @paoloricciuti should be added here too, he has been very involved since joining Mainmatter (and he's a big name in the Svelte community so could also be good for publicity)

**Comment by beerinho** [comment]
>@beerinho @zeppelin @pichfl @paoloricciuti do you have ideas for a good header image?

I asked the team this morning but no one could really think of a good header image for svelte. The one that came up would just be a generic photo of a workshop - someone standing in front of a group

File: src/workshops/svelte-and-sveltekit.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      enhancement with forms. We cover how to send data to an API, how to
+      validate forms, what server folders are, and give a brief introduction
+      into hooks. We close by looking into how to implement authentication.
+leads:
+  - name: Daniel Beer
+    title: Senior Frontend Engineer
+    handle: beerinho
+    authorHandle: beerinho
+    image: /assets/images/authors/beerinho.jpg
+    bio: >
```
Comment (KevinBongart) [inline src/workshops/svelte-and-sveltekit.md:72]: @beerinho Can you complete you bio directly or give me something you'd like to see here?

File: src/workshops/svelte-and-sveltekit.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      …
+  - name: Gabor Babicz
+    title: Senior Frontend Engineer
+    handle: zeppelin
+    authorHandle: zeppelin
+    image: /assets/images/authors/zeppelin.jpg
+    bio: >
```
Comment (KevinBongart) [inline src/workshops/svelte-and-sveltekit.md:81]: @zeppelin Can you complete you bio directly or give me something you'd like to see here?

File: src/workshops/svelte-and-sveltekit.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      …
+  - name: Florian Pichler
+    title: Senior Frontend Engineer
+    handle: pichfl
+    authorHandle: pichfl
+    image: /assets/images/authors/pichfl.jpg
+    bio: >
+      Florian has been solving frontend challenges for over 15 years, with all
+      tools that became available. He has built and coached teams of all sizes.
+---
```
Comment (KevinBongart) [inline src/workshops/svelte-and-sveltekit.md:94]: @paoloricciuti Can you add your info in here directly, or give me a profile pic and a bio that you'd like to see here?

**Comment by beerinho** [comment]
Hey @KevinBongart, @paoloricciuti, @pichfl and I have added our bios on the [Notion page](https://www.notion.so/mainmatter/Create-workshop-overview-page-for-Mainmatter-com-2635fede2e224edea5d9995a541454bd?pvs=4)

File: src/workshops/svelte-and-sveltekit.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      The final stage of the workshop teaches how to implement progressive
+      enhancement with forms. We cover how to send data to an API, how to
+      validate forms, what server folders are, and give a brief introduction
+      into hooks. We close by looking into how to implement authentication.
+leads:
+  - name: Paolo Ricciuti
+    title: Senior Frontend Engineer
+    handle: paoloricciuti
+    authorHandle: paoloricciuti
+    image: /assets/images/authors/paoloricciuti.jpg
```
Comment (marcoow) [inline src/workshops/svelte-and-sveltekit.md:61]: @paoloricciuti feel free to change the image if you'd rather use a different one

Comment (paoloricciuti) [reply src/workshops/svelte-and-sveltekit.md:61]: This one is fine :)

**Pull request merged**